Tests/TextToSpeechV1Tests/TextToSpeechTests.swift

```diff
@@ -100,41 +100,27 @@ class TextToSpeechTests: XCTestCase {
     func deleteCustomizations() {
         let description = "Delete all customizations."
         let expectation = self.expectation(description: description)
-        textToSpeech.listVoiceModels(failure: failExceptLitePlan(expectation: expectation)) { voiceModels in
+        textToSpeech.listVoiceModels() {
+            response, error in
+            if let error = error {
+                if !error.localizedDescription.contains(self.litePlanMessage) {
+                    XCTFail("Unexpected error response from service: \(error)")
+                }
+                expectation.fulfill()
+                return
+            }
+            guard let voiceModels = response?.result else {
+                XCTFail("Missing result value")
+                return
+            }
             for voiceModel in voiceModels.customizations {
-                self.textToSpeech.deleteVoiceModel(customizationID: voiceModel.customizationID) { }
+                self.textToSpeech.deleteVoiceModel(customizationID: voiceModel.customizationID) { _,_ in }
             }
             expectation.fulfill()
         }
         waitForExpectations()
     }
 
-    /** Fail false negatives. */
-    func failWithError(error: Error) {
-        XCTFail("Positive test failed with error: \(error)")
-    }
-
-    /** Fail false positives. */
-    func failWithResult<T>(result: T) {
-        XCTFail("Negative test returned a result.")
-    }
-
-    /** Fail false positives. */
-    func failWithResult() {
-        XCTFail("Negative test returned a result.")
-    }
-
-    /** Fail unless error is due to Lite Plan */
-    func failExceptLitePlan(expectation: XCTestExpectation,
```

Comment:
Why get rid of this function? It saved a lot of code duplication.

Reply:
Nevermind, you told me why you prefer to see the `XCTFail()` occur within the test that triggers it.